### PR TITLE
chore(#364): Link to bugs tracker and repository

### LINKED ISSUE #364: Link to bug tracker and repository in package.json
Socket.dev is lowering our score because our package.json is missing some of the links they expect: the website, the issue tracker, and the repository. Let's go through all our package.jsons and add the required fields.

### PR DESCRIPTION
**Pull request by Ashvith10**
Fixes #364 

PR is in draft to discuss about the repo links for internal packages.

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -40,5 +40,11 @@
         "dist/**/*",
         "package.json",
         "README.md"
-    ]
+    ],
+    "bugs": { "url" : "https://github.com/doug-wade/tybalt/issues" },
```
Comment (doug-wade) [inline packages/core/package.json:44]: I think this field is meant to be a link to the currently open bugs for this node module, so I think we can link to the tag we maintain for this package: https://github.com/doug-wade/tybalt/issues?q=is%3Aissue+is%3Aopen+label%3Acore

File: packages/core/package.json [hunk 2]
```diff
@@ -40,5 +40,11 @@
...
+    "homepage": "https://tybalt.org",
```
Comment (doug-wade) [inline packages/core/package.json:45]: I think we can link to the page specific to each public package's page: https://tybalt.org/pages/core/

**Review by doug-wade — CHANGES_REQUESTED** [review]

File: packages/eleventy-plugin-vanilla-example/package.json [hunk 3]
```diff
@@ -19,5 +19,11 @@
         "@11ty/eleventy": "^2.0.1",
         "@tybalt/esbuild-plugin": "workspace:^",
         "rimraf": "^5.0.1"
+    },
+    "bugs": { "url" : "https://github.com/doug-wade/tybalt/issues" },
```
Comment (Ashvith10) [inline packages/eleventy-plugin-vanilla-example/package.json:23]: @doug-wade what about `eleventy-plugin-vanilla-example`? There's no such label or homepage...

File: packages/example/package.json [hunk 4]
```diff
@@ -29,5 +29,14 @@
         "esbuild": "^0.16.14",
         "esbuild-jest": "^0.5.0",
         "jest-environment-jsdom": "^29.6.1"
+    },
+    "bugs": {
+        "url" : "https://github.com/doug-wade/tybalt/issues?q=is%3Aissue+is%3Aopen+label%3Aexample"
+    },
+    "homepage": "https://tybalt.org/pages/example",
```
Comment (doug-wade) [inline packages/example/package.json:36]: I don't think this page exists; probably https://tybalt.org/ works

File: packages/test-utils-lit-example/package.json [hunk 5]
```diff
@@ -23,5 +23,14 @@
     },
     "dependencies": {
         "lit": "^2.8.0"
+    },
+    "bugs": {
+        "url" : "https://github.com/doug-wade/tybalt/issues?q=is%3Aissue+is%3Aopen+label%3Atest-utils-lit-example"
+    },
+    "homepage": "https://tybalt.org/pages/test-utils-lit-example",
```
Comment (doug-wade) [inline packages/test-utils-lit-example/package.json:30]: I don't think this page exists -- ttps://tybalt.org/pages/test-utils ?

**Review by doug-wade — CHANGES_REQUESTED** [review]

**Comment by doug-wade** [comment]
We needed this for the release, so I landed it in https://github.com/doug-wade/tybalt/pull/438

**Pull request closed without merging**